### DISCUSSION TITLE
I only want Frigate events to be recorded but it seems like recording is 24*7

### DISCUSSION THREAD

**Discussion by tomhusband** [blakeblackshear/frigate#5693] [Ask A Question]
I have Home assistant in Proxmox and I'm wondering if my frigate config is off. If I view the Frigate files in Media I have hundreds of recordings that keep piling up until there's no room left. Could the record: portion in config be in the wrong place?

I'm having to ssh into my driveway_cam and delete the recordings. That certainly isn't right.

Any hints greatly appreciated.
Here's my config:
https://pastebin.com/uKTVJWNu

**Comment by blakeblackshear** [comment]
That looks correct to me. I would look at the logs for clues.

You can also enable debug logging for the record process
```yaml
logger:
  default: info
  logs:
    frigate.record: debug
```

**Reply by tomhusband** [reply]
Thanks, I changed it to debug so we'll see what come up. So maybe the problem is outside of the config file?

**Reply by blakeblackshear** [reply]
You should only see 10s segments that contain active objects for the last 4 days with your config.

**Comment by blakeblackshear** [comment]
Also you can wrap logs and configs in triple back ticks to get it to format correctly: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

**Reply by tomhusband** [reply]
Thanks again, I missed that.
